l10n_it_vat_registries/report/report_registro_iva.xml

```diff
@@ -223,7 +223,7 @@
                                         </tr>
                                     </thead>
                                     <t
-                                            t-foreach="total_used_taxes"
+                                            t-foreach="total_used_taxes.sorted(key='sequence')"
```

Comment:
Grazie della PR!

```suggestion
                                            t-foreach="total_used_taxes.sorted()"
```
Senza parametri viene preso l'ordinamento definito nel model (vedi https://github.com/odoo/odoo/blob/cb996aefb4aeeac9c41359ead523d52c66d21e29/odoo/models.py#L5387-L5389) che se ho ben capito dovrebbe essere quello che si vuole ottenere da questa PR.

Specificando l'ordinamento qui per `sequence` si rischia di doverlo cambiare quando verrà modificato l'ordinamento nel model che attualmente è per `sequence,id`:  https://github.com/odoo/odoo/blob/c8c68a07199124091e8d49946a63fb8c705c2417/addons/account/models/account_tax.py#L40

Reply:
Grazie per la review!

Il problema portato dal cliente era l'impossibilità di controllare l'ordine del riepilogo, oltre al fatto che cambiava da stampa a stampa. Quello che vogliono otterene è una corrispondenza 1:1 con le stampe che fa il loro commercialista col suo sw, compreso l'ordine con cui le imposte vengono riepilogate!

Quindi in realtà, il problema non è solo avere un ordinamento, e l'ordinamento di default, specie se prendiamo in considerazione l'ipotesi (oggettivamente abbastanza improbabile) che possa cambiare, non è quello ideale: quello ideale è quello impostato da interfaccia in fase di configurazione (e quindi basato su sequence, e non su altro).

In altre parole, usando quello di default avremmo il problema opposto, mettere esplicitamernte sequence come key se dovesse cambiare.

Sebbene probabilmente ai più basterebbe un ordinamento qualunque, basta che sia coerente da stampa a stampa, l'idea di poterlo controllare precisamente da configurazione non è sbagliata.

Reply:
Ah quindi non importa che la visualizzazione nel backend (determinata da https://github.com/odoo/odoo/blob/c8c68a07199124091e8d49946a63fb8c705c2417/addons/account/models/account_tax.py#L40) sia coerente con la visualizzazione nel report?
Se mi confermi che è così allora va bene; secondo me può risultare inaspettato ma non lo vedo come un problema.

Per avere un ordinamento univoco aggiungerei comunque `sorted('id')` altrimenti a parità di `sequence` (ad esempio quando le crei hanno tutte lo stesso default) potresti avere report con le imposte in ordini diversi.

Reply:
> Ah quindi non importa che la visualizzazione nel backend (determinata da https://github.com/odoo/odoo/blob/c8c68a07199124091e8d49946a63fb8c705c2417/addons/account/models/account_tax.py#L40) sia coerente con la visualizzazione nel report?

Non è quello il problema riportato. Alle fine stiamo un po' discutendo del sesso degli angeli, non è che sia poi così probabile che l'ordinamento di default cambi, per cui anche mettere .sorted() e basta al 99,99% non farà mai alcuna differenza. Era più per spiegare perché avevo fatto con sequence esplicita.